server/notification-providers/signl4.js

```diff
@@ -0,0 +1,85 @@
+const NotificationProvider = require("./notification-provider");
+const axios = require("axios");
+const { Liquid } = require("liquidjs");
+const { UP, DOWN } = require("../../src/util");
+
+class SIGNL4 extends NotificationProvider {
+
+    name = "SIGNL4";
+
+    /**
+     * @inheritdoc
+     */
+    async send(notification, msg, monitorJSON = null, heartbeatJSON = null) {
+        let okMsg = "Sent Successfully.";
```

Comment:
please prefer const where possible. (this communicate intentions a bit clearer, I know that this is not effective in stopping bugs)

```suggestion
        const okMsg = "Sent Successfully.";
```

Reply:
OK, thanks.